src/main/java/uk/gov/pay/api/validation/SearchValidator.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.pay.api.validation;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+
+import static org.eclipse.jetty.util.StringUtil.isNotBlank;
+
+
+public abstract class SearchValidator {
+
+    protected static void validatePageIfNotNull(String pageNumber, List<String> validationErrors) {
+        if (isNotBlank(pageNumber) && (!StringUtils.isNumeric(pageNumber) || Integer.valueOf(pageNumber) < 1)) {
```

Comment:
There is a NumericValidator in PublicApi for this. If you ask why we are not using isNumeric, help yourself https://github.com/alphagov/pay-publicapi/pull/267#discussion_r220148191